### DISCUSSION TITLE
TuYA TS0601 Thermostat valve only opened from 35%-65%

### DISCUSSION THREAD

**Discussion by d3j4vuX** [Koenkk/zigbee2mqtt#15752] [Devices]
Hello friends,
I have a problem with my thermostats from TuYa. My setup consists of a RasberryPi 4 and a Zigbee Sonoff Stick V3. I bought 4x the TS0601 and have the same problem on all radiators. When the valve is 5-30% open according to HomeAssistant, no water flows through the radiator. Only from 35%-60% the valve is opened and at 65% it is already completely open. This is always too much or too little heated because the thermostat when heating, for example, assumes 65% but is actually heated with 100%.
Is this a known problem? I have not been able to find anything on the Internet.
When I open and close the valve by hand I can hear that even at the smallest opening of the valve water flows.

thanks

**Comment by freeDom-** [comment]
This sounds like the issue I had after installing the thermostat my heater did not stopped heating no matter what temperature it was. The issue is well documented here: https://community.home-assistant.io/t/thermostat-wont-stop-heating-tuya-ts0601/476340

In Short:
Go to the settings and press OK 10 times until you see an 'A' flashing
Change the value from 0 to 1
